common/lib/xmodule/xmodule/templates/problem/circuitschematic.yaml

```diff
@@ -60,24 +73,17 @@ data: |
       else:
         correct = ['incorrect']
       </answer>
-      </schematicresponse>
-
-      <solution>
-          <div class="detailed-solution">
-              <p>Explanation</p>
-              <p>
-              You can form a voltage divider that evenly divides the input
-              voltage with two identically valued resistors, with the sampled
-              voltage taken in between the two.
-              </p>
-              <p><img src="/static/images/voltage_divider.png" alt=""/></p>
-              <p>
-              You can form a simple high-pass filter without any further
-              constraints by simply putting a resistor in series with a
-              capacitor. The actual values of the components do not really
-              matter in this problem.
-              </p>
-              <p><img src="/static/images/high_pass_filter.png" alt=""/></p>
-          </div>
-      </solution>
+            <solution>
+                <div class="detailed-solution">
+                    <p>Explanation</p>
+                    <p>
+                        You can form a simple high-pass filter without any further
+                        constraints by simply putting a resistor in series with a
+                        capacitor. The actual values of the components do not really
+                        matter in this problem.
+                    </p>
+                    <p><img src="/static/images/high_pass_filter.png" alt=""/></p>
+                </div>
+            </solution>
+        </schematicresponse>
```

Comment:
@muzaffaryousaf I have moved only those tags inside which are most relevant to responsetype. The `<p>` tags in `circuitschematic.yaml` are problem wide description.